examples/components/MNIST/aggregatemodelweights/run.py

```diff
@@ -0,0 +1,132 @@
+import os
+import argparse
+import logging
+import sys
+
+import torch
+from torch import nn
+from torchvision import models
+
+
+# Set logging to sys.out
+logger = logging.getLogger(__name__)
```

Comment:
Could you add this in the main() function?

Reply:
(or at least in the main block after if __name__ ...)

Reply:
Moved the logger to the "if" block